core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConsoleHandler.java

```diff
@@ -37,7 +37,7 @@ public void install() {
         public void handleInput(int[] keys) {
             if (disabled) {
```

Comment:
Should it be changed to paused?

Reply:
Maybe, but there is a different internal 'paused' that happens when a dev mode hot reload is going on (so you don't have the tests potentially slowing down user interaction).